deploy/deploy-wmagent.sh

```diff
@@ -285,12 +293,15 @@ echo -e "\n*** Applying agent patches ***"
 if [ "x$PATCHES" != "x" ]; then
   cd $CURRENT_DIR
   for pr in $PATCHES; do
-    wget -nv https://github.com/dmwm/WMCore/pull/$pr.patch -O - | patch -d apps/wmagent/lib/python2*/site-packages/ -p 3
+    wget -nv https://github.com/dmwm/WMCore/pull/$pr.patch -O - | patch -d apps/${RPM_NAME}/lib/python2*/site-packages/ -p 3
```

Comment:
I am pretty sure you have already tested it, but just my curiosity: Why hard coded path leading to `python2` libraries even though here we provide the option to deploy a python3 version of the rpm package

Reply:
Good catch! This needs to be fixed.